jdaviz/configs/default/plugins/model_fitting/model_fitting.py

```diff
@@ -18,18 +18,19 @@
 from jdaviz.core.registries import tray_registry
 from jdaviz.core.template_mixin import TemplateMixin
 from jdaviz.configs.default.plugins.model_fitting.fitting_backend import fit_model_to_spectrum
-from jdaviz.configs.default.plugins.model_fitting.initializers import initialize, model_parameters
+from jdaviz.configs.default.plugins.model_fitting.initializers import (MODELS,
```

Comment:
I don't know how I feel about modifying a global dictionary from another module. Probably good enough for now but if we see weird or hard to debug behaviors, especially given this plugin seems to be called by 3-4 different vizes, we might want to rethink this.

Reply:
It isn't ever modified, we just need access to the string -> class mapping in both places and I thought this was slightly better than copying the dictionary in both files.  Would it be cleaner to make it a function so that it couldn't be modified?

Reply:
Ah, sorry. I must have misread the code... Doesn't look like Python has built-in read-only `dict`. Ah well. No need to overcomplicate the matter. You can ignore this comment.